walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -14,10 +14,42 @@
 
 package com.google.sps;
 
-import java.util.Collection;
+import java.sql.Time;
+import java.util.*;
 
 public final class FindMeetingQuery {
   public Collection<TimeRange> query(Collection<Event> events, MeetingRequest request) {
-    throw new UnsupportedOperationException("TODO: Implement this method.");
+    SortedSet<TimeRange> timeRangesSorted = new TreeSet<TimeRange>(TimeRange.ORDER_BY_START);
```

Comment:
For readability, either change variable name to `takenTimeRangesSorted` or add a comment describing what the set holds.